manager/container.go

```diff
@@ -572,9 +579,116 @@ func (c *containerData) updateStats() error {
 	if statsErr != nil {
 		return statsErr
 	}
+	if nvidiaStatsErr != nil {
+		return nvidiaStatsErr
+	}
 	return customStatsErr
 }
 
+// updateNvidiaStats updates the stats for NVIDIA GPUs (if any) attached to the container.
+func (c *containerData) updateNvidiaStats(stats *info.ContainerStats) error {
+	if !nvmlInitialized || len(nvidiaDevices) == 0 {
+		return nil
+	}
+	nvidiaMinorNumbers := c.getNvidiaMinorNumbers()
+	for _, minor := range nvidiaMinorNumbers {
+		device := nvidiaDevices[minor]
```

Comment:
as your new commit, the nvidiaDevices will be updated every 5 minutes. should we sync to lock it up here as well?